### ISSUE TITLE
Implement asset packer tool

### ISSUE THREAD

**Issue by brokenprogrammer** [Realbrokenprogrammer/game#52]
This is a separate application from the game so what I suggest is to create a new folder in the Game's root directory called "Tools" that will contain subfolders with the different tools we develop for the game. 

The tool to be implemented for this issue is supposed to pack assets into the format we should have specified after #17 is merged into master.

A good idea will also be to document the asset file structure and store it within the knowledge base.
